### ISSUE TITLE
[Object] Default Styles for Text Panel

### ISSUE THREAD

**Issue by brusch** [pimcore/pimcore#3338]
#### Alerts
- danger
- success
- warning
- info

#### Others
- headlines
- lists

**Comment by dpfaffenbauer** [comment]
Docs for that would be good.

**Comment by weisswurstkanone** [comment]
isn't there a hint in the class editor ?
see the change in bundles/AdminBundle/Resources/public/js/pimcore/object/classes/layout/text.js

**Comment by brusch** [comment]
Yep, the same text as above ... it's right there where it is used, so imho it doesn't need to be additionally  mentioned in the docs ... so I thought at least ;)

**Comment by dpfaffenbauer** [comment]
Would still be good to have that here, so developer knows that it is possible:

https://pimcore.com/docs/5.x/Development_Documentation/Objects/Object_Classes/Layout_Elements/Dynamic_Text_Labels.html
